compiler/tests/generated/alias_key_types.h

```diff
@@ -45,9 +45,16 @@ namespace test
 #endif
         
         
-#ifndef BOND_NO_CXX11_DEFAULTED_FUNCTIONS
+#if !defined(BOND_NO_CXX11_DEFAULTED_MOVE_CTOR)
         // Compiler generated operator= OK
         foo& operator=(const foo&) = default;
+        foo& operator=(foo&&) = default;
+#else
+        foo& operator=(foo other)
```

Comment:
What is the result of `std::is_(nothrow_)move_assignable` for a type that just has this kind of assignment operator?

Reply:
The `std::is_move_assignable` will of course be `true`, and the `std::is_nothrow_move_assignable` will be `false`. Note, that STL containers gained `noexcept` only in C++17 for their move-assignment operators.